_includes/data/additional_resources.html

```diff
@@ -3,7 +3,7 @@
       <li><a href="http://build.fhir.org/ig/HL7/VhDir/bulk-data.html" target="_blank" rel="noopener" class="in-text__link"> Bulk FHIR specification </a></li>
       <li><a href="https://bluebutton.cms.gov/developers/" target="_blank" rel="noopener" class="in-text__link"> Beneficiary FHIR Data Server (BFD)/ Blue Button API </a></li>
       <li><a href="https://bluebutton.cms.gov/assets/ig/index.html" target="_blank" rel="noopener" class="in-text__link"> Beneficiary FHIR Data Server (BFD)/ Blue Button Implementation Guide </a></li>
-      <li><a href="https://www.json.org/json-en.html" target="_blank" rel="noopener" class="in-text__link"> Intro to JSON Format </a> and <a href="http://ndjson.org/" target="_blank" rel="noopener" class="in-text__link"> NDJSON.org </a></li>
+      <li><a href="https://www.json.org/json-en.html" target="_blank" rel="noopener" class="in-text__link"> Intro to JSON Format </a> and <a href="https://github.com/ndjson/ndjson-spec/" target="_blank" rel="noopener" class="in-text__link"> ndjson-spec </a></li>
```

Comment:
same as previous suggestion

```suggestion
      <li>Intro to the <a href="https://www.json.org/json-en.html" target="_blank" rel="noopener" class="in-text__link">JSON Format</a> and <a href="https://github.com/ndjson/ndjson-spec/" target="_blank" rel="noopener" class="in-text__link">NDJSON</a></li>
```